yellow-paper/docs/public-vm/gen/_InstructionSet.mdx

```diff
@@ -0,0 +1,1240 @@
+	<td><Markdown>Unsigned division (a / b)</Markdown></td>
+	<td><Markdown>96</Markdown></td>
+	<td><CodeBlock language="jsx">{
+		`M[dstOffset] = M[aOffset] / M[bOffset]`
+	}</CodeBlock></td>
+</tr>
+<tr>
+	<td style={{'text-align': 'center'}}>0x03</td>	<td style={{'text-align': 'center'}}><a id='isa-table-eq'/><Markdown>[`EQ`](#isa-section-eq)</Markdown></td>
+	<td><Markdown>Equality check (a == b)</Markdown></td>
+	<td><Markdown>96</Markdown></td>
+	<td><CodeBlock language="jsx">{
+		`M[dstOffset] = M[aOffset] == M[bOffset] ? 1 : 0`
+	}</CodeBlock></td>
+</tr>
+<tr>
+	<td style={{'text-align': 'center'}}>0x04</td>	<td style={{'text-align': 'center'}}><a id='isa-table-lt'/><Markdown>[`LT`](#isa-section-lt)</Markdown></td>
+	<td><Markdown>Less-than check (a &lt; b)</Markdown></td>
+	<td><Markdown>96</Markdown></td>
+	<td><CodeBlock language="jsx">{
+		`M[dstOffset] = M[aOffset] < M[bOffset] ? 1 : 0`
+	}</CodeBlock></td>
+</tr>
+<tr>
+	<td style={{'text-align': 'center'}}>0x05</td>	<td style={{'text-align': 'center'}}><a id='isa-table-lte'/><Markdown>[`LTE`](#isa-section-lte)</Markdown></td>
+	<td><Markdown>Less-than-or-equals check (a &lt;= b)</Markdown></td>
+	<td><Markdown>96</Markdown></td>
+	<td><CodeBlock language="jsx">{
+		`M[dstOffset] = M[aOffset] <= M[bOffset] ? 1 : 0`
+	}</CodeBlock></td>
+</tr>
+<tr>
+	<td style={{'text-align': 'center'}}>0x06</td>	<td style={{'text-align': 'center'}}><a id='isa-table-and'/><Markdown>[`AND`](#isa-section-and)</Markdown></td>
+	<td><Markdown>Bitwise AND (a & b)</Markdown></td>
+	<td><Markdown>96</Markdown></td>
+	<td><CodeBlock language="jsx">{
+		`M[dstOffset] = M[aOffset] AND M[bOffset]`
+	}</CodeBlock></td>
+</tr>
+<tr>
+	<td style={{'text-align': 'center'}}>0x07</td>	<td style={{'text-align': 'center'}}><a id='isa-table-or'/><Markdown>[`OR`](#isa-section-or)</Markdown></td>
+	<td><Markdown>Bitwise OR (a | b)</Markdown></td>
+	<td><Markdown>96</Markdown></td>
+	<td><CodeBlock language="jsx">{
+		`M[dstOffset] = M[aOffset] OR M[bOffset]`
+	}</CodeBlock></td>
+</tr>
+<tr>
+	<td style={{'text-align': 'center'}}>0x08</td>	<td style={{'text-align': 'center'}}><a id='isa-table-xor'/><Markdown>[`XOR`](#isa-section-xor)</Markdown></td>
+	<td><Markdown>Bitwise XOR (a ^ b)</Markdown></td>
+	<td><Markdown>96</Markdown></td>
+	<td><CodeBlock language="jsx">{
+		`M[dstOffset] = M[aOffset] XOR M[bOffset]`
+	}</CodeBlock></td>
+</tr>
+<tr>
+	<td style={{'text-align': 'center'}}>0x09</td>	<td style={{'text-align': 'center'}}><a id='isa-table-not'/><Markdown>[`NOT`](#isa-section-not)</Markdown></td>
+	<td><Markdown>Bitwise NOT (inversion)</Markdown></td>
+	<td><Markdown>72</Markdown></td>
+	<td><CodeBlock language="jsx">{
+		`M[dstOffset] = NOT M[aOffset]`
+	}</CodeBlock></td>
+</tr>
+<tr>
+	<td style={{'text-align': 'center'}}>0x0a</td>	<td style={{'text-align': 'center'}}><a id='isa-table-shl'/><Markdown>[`SHL`](#isa-section-shl)</Markdown></td>
```

Comment:
sequential opcode naming is nice, however it makes sense to have buffers to group like opcodes, such that new ones can be injected into like places

Reply:
not sure exactly what you mean